### ISSUE TITLE
Hokify Job Bot

### ISSUE THREAD

**Issue by maxstolze** [researchstudio-sat/webofneeds#2370]
Create an Hokify Job Bot:
- [x] Fetch jobdata
- [x] Create and publish job needs
- [x] React to user requests
- [x] Get location data
- [x] Set fetch/update intervall
- [ ] Filter out contact info
- [ ] Parse job type field

requires #2520
because the structure of the need creation has been changed, to reduce the pain of merging/rebasing the required PR should be merged before hand

**Comment by quasarchimaere** [comment]
moving from in-progress to ready since assignment was empty

**Comment by maxstolze** [comment]
So far the issue seemed to be solved and the changes and adjustments of the job bot could be an own issue for future work.
